### ISSUE TITLE
filtering simliarly as done for stored data sets from 10x

### ISSUE THREAD

**Issue by anupparikh** [biolab/orange3-single-cell#69]
we have a few data sets that use dispersion as a metric to filter genes. 
Can I do such filtering using the current filter widget? If not is this method worth implementing?

**Comment by BlazZupan** [comment]
Dispersion-based filtering is already implemented in Score Genes. But it does not show any visualization. Is that ok, or do you think it is better that we move all the gene scoring methods to Filter widget and then remove Score Genes?

**Comment by lanzagar** [comment]
We now have 2 possible ways to filter genes/cells (after #133). We can think of adding a third. But we should try to avoid too much complexity and overlap between the Score Genes and Filter widgets.

I am thinking we could move the most general unsupervised metrics to Orange's Rank widget (e.g. mean, variance) and move single-cell oriented things to Filter. And then remove Score Genes which is a duplicate of Rank with a couple of extra metrics.
